### ISSUE TITLE
Is Bybit exange supported?

### ISSUE THREAD

**Issue by LarryEitel** [Drakkar-Software/OctoBot#1897]
Didn't find any reference to this exchange. Thank You.

**Comment by techfreaque** [comment]
its coming soon, its already almost done

**Comment by LarryEitel** [comment]
Thank you. Would you mind posting here when it is ready?

**Comment by techfreaque** [comment]
yeah we'll close all related future requests, so if you're subscribed you'll get notified

**Comment by blueyed** [comment]
Unfortunately not anymore.
